Source/Core/Core/HW/WiimoteEmu/Extension/Drums.cpp

```diff
@@ -118,23 +118,21 @@ void Drums::Update(const DesiredExtensionState& target_state)
     desired_state.stick_y = STICK_CENTER;
     desired_state.buttons = 0;
     desired_state.drum_pads = 0;
-    desired_state.softness = 7;
+    desired_state.softness = 0x7F;
```

Comment:
Github isn't letting me comment on the specific line, but the definition of this member-variable has a comment about the size that needs updating, from 3 to 7.